src/train_tracker.py

```diff
@@ -0,0 +1,221 @@
+"""列車捕捉モジュール.
+
+列車を捕捉する
+@author: desty505 miyashita64
+"""
+
+import cv2
+import numpy as np
+from camera_interface import CameraInterface
+
+
+class TrainTracker:
+    """列車を捕捉するクラス."""
+
+    def __init__(self, camera_id: int) -> None:
+        """コンストラクタ.
+
+        Args:
+            camera_id (int): カメラID
+        """
+        self.camera = CameraInterface(camera_id)
+        self.diff_border = 20       # 動体検知におけるフレームの差の閾値
+        self.observe_rect_points = [(0, 0), (0, 0)]
+
```

Comment:
```suggestion
    def __del__(self) -> None:
        """デストラクタ."""
```
俺がいらないといったのは、`del self.camera`であって、デストラクタ自体をいらないとは言ってない。上のやつはいる。もしデストラクタ自体いらないと判断したならば、camera_system.pyの43行目の`del tt`は消すべき

Reply:
ごめん。いらないね。なくてもdelできるらしいですね。